idea/testData/inspectionsLocal/replaceWithOperatorAssignment/validAddition.kt

```diff
@@ -0,0 +1,4 @@
+fun foo() {
```

Comment:
This test does not test your change yet. You should add comment above (to both this file and after file):

```
// FIX: expected fix text
```

to check that fix text is correct.